src/routes/messagesRoutes.ts

```diff
@@ -1,16 +1,22 @@
-import express, { Response } from "express";
-import {
-  getAllMessages,
-  getMessageByUserId,
-  createMessage,
-} from "../controllers/messagesController";
-
-import { logBody } from "../middlewares/messageMiddleware";
-
+import express from 'express';
+import { messageController } from '../controllers';
+import { validateReqBodySchema } from '../middlewares/validateReqBodySchema';
+import { validateReqParamSchema } from '../middlewares/validateReqParamSchema';
+import { logBody } from '../middlewares/messageMiddleware';
+import { userIdParamSchema, messageSchema } from '../schemas';
 const router = express.Router();
 
-router.get("/", getAllMessages);
-router.get("/:userId", getMessageByUserId);
-router.post("/", logBody, createMessage);
+router.get('/', messageController.getAllMessages);
+router.get(
+  '/:userId',
+  validateReqParamSchema(userIdParamSchema),
+  messageController.getUserMessages
+);
+router.post(
+  '/',
+  logBody,
+  validateReqBodySchema(messageSchema),
+  messageController.createMessage
+);
```

Comment:
maybe we should look at adding some lint/prettier rules? what do you think?

Reply:
It would be great, I will try to do it.